site/profile/manifests/ts/dco.pp

```diff
@@ -4,6 +4,23 @@
 class profile::ts::dco {
   $user = 'dco'
 
+  file { "/home/${user}/docker_tmp":
+    ensure => directory,
+    mode   => '0777',
+    owner  => $user,
+    group  => $user,
+  }
+
+  vcsrepo { "/home/${user}/docker-compose-admin":
```

Comment:
The spec for this change is currently part of ad0962ab1be5fcc928f1cc63e977e0065050f37d and should be moved into this commit.